packages/block-library/src/image/edit.js

```diff
@@ -525,7 +537,6 @@ class ImageEdit extends Component {
 									showRightHandle = true;
 								}
 							}
-							/* eslint-enable no-lonely-if */
```

Comment:
This shouldn't be changed. We need to reenable any ESLint immediately after it is no longer necessary to be disabled.